storage/innobase/include/univ.i

```diff
@@ -485,6 +485,8 @@ typedef uint32_t ib_uint32_t;
 /* Type used for all log sequence number storage and arithmetics */
 typedef	ib_uint64_t		lsn_t;
 
+typedef ib_uint64_t		ullong;
+
```

Comment:
It's not necessary (`ib_uint64_t` may be used instead), but to comply code consistency I found it useful. The type `ulint` used in code that I cut&pasted does not guarantee 64 bits (at least 32 bits in size).